src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
 public class SalaryInfo {
+    private VerifyData verifyData = new VerifyData();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        SimpleDate startDate = new SimpleDate(dateFrom);
+        SimpleDate endDate = new SimpleDate(dateTo);
```

Comment:
Make sure that the `SimpleDate` class is implemented and supports the operations used here, such as parsing dates and checking if a date is within a range.